packages/fuels/tests/contracts/revert_transaction_error/src/main.sw

```diff
@@ -1,15 +1,15 @@
 contract;
 
 abi MyContract {
-    fn make_transaction_fail(input: u64) -> u64;
+    fn make_transaction_fail(fail: bool) -> u64;
 }
 
 impl MyContract for Contract {
-    fn make_transaction_fail(input: u64) -> u64 {
-        // Work around DCE and revert
-        if true {
-            revert(input);
+    fn make_transaction_fail(fail: bool) -> u64 {
+        if fail {
```

Comment:
The previous comment mentioned working around the `DCE`. Does this change still work around the `DCE`, whatever that is?

Reply:
FYI, DCE seems to mean [dead code elimination](https://en.wikipedia.org/wiki/Dead-code_elimination).

Reply:
I changed the input to be a boolean so we cat decide what behavior we want.